docs/api/flink/Function.md

```diff
@@ -993,6 +993,31 @@ FROM polygondf
 !!!note
     The detailed EPSG information can be searched on [EPSG.io](https://epsg.io/).
 
+## ST_Translate
+Introduction: Returns the input geometry with its X, Y and Z coordinates (if present in the geometry) translated by deltaX, deltaY and deltaZ (if specified)
+
+If the geometry is 2D, and a deltaZ parameter is specified, no change is done to the Z coordinate of the geometry and the resultant geometry is also 2D.
+
+If the geometry is empty, no change is done to it.
+
+If a geometry collection is given, all geometries of the collection are individually translated.
```

Comment:
This should apply to all geometries such as multipoint, multipolygon, multilinestring, and geometrycollection. Not only geometrycollection.